miscellaneous-commands/premium.md

```diff
@@ -1,6 +1,6 @@
 # premium
 
-<TODO: describe>
+This command links you to the help doccument where you can [learn more about MoonlightBot Premium.](/support/premium.md)
```

Comment:
This command behaves much differently when:

- you have no premium
- you have premium
- the server has premium, but you personally don't have it